packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md

```diff
@@ -30,7 +30,7 @@ The difference between building these two types of assertions is the algorithm a
 
 Which JWT type you use depends on the client authentication method configured in your OAuth 2.0 client application. See [Client Authentication Methods](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/client-auth/#client-authentication-methods) for more information on the supported methods and when to use them.
 
-> **Note:** To create a client application and specify either the `client_secret_jwt` or `private_key_jwt` authentication method, see the [Add OAuth 2.0 Client Application](/docs/reference/api/apps/#add-oauth-2-0-client-application) API reference section. To change the client authentication method of an existing app, see the [Update the Client Authentication Method](/docs/reference/api/apps/#update-the-client-authentication-method) API reference section.
+> **Note:** To create a client application and specify either the `client_secret_jwt` or `private_key_jwt` authentication method, see the [Add OAuth 2.0 Client Application](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/createApplication!path=4/name&t=request) API reference section. To change the client authentication method of an existing app, see the [Update the Client Authentication Method](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/oauthClient/token_endpoint_auth_method&t=request) API reference section.
```

Comment:
The Add OAuth 2.0 Client Application no longer exists.

Update the Client Authentication Method - where is this section?

```suggestion
> **Note:** To create a client app and specify either the `client_secret_jwt` or `private_key_jwt` authentication method, see the [parameter details](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/createApplication!path=4/name&t=request) of the Apps API reference section. To change the client authentication method of an existing app, see the [Update the Client Authentication Method](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/oauthClient/token_endpoint_auth_method&t=request) API reference section.
```